From a8b173d725f0c3baf6415fc109f58c2cb7e48537 Mon Sep 17 00:00:00 2001
From: Michael Orlitzky <michael@orlitzky.com>
Date: Tue, 2 Sep 2025 00:52:26 -0400
Subject: [PATCH] lib/package.gi: don't load PackageManager by default

That is, we revert:

  https://github.com/gap-system/gap/commit/ea07baa0

We don't package the GAP PackageManager, and would not want users to
use it if we did.
---
 lib/package.gi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/package.gi b/lib/package.gi
index 8916dc3..6c870c5 100644
--- a/lib/package.gi
+++ b/lib/package.gi
@@ -1948,7 +1948,7 @@ For backwards compatibility, the default lists most of packages \
 that were autoloaded in &GAP; 4.4 (add or remove packages as you like)."
     ],
   default:= [ "autpgrp", "alnuth", "crisp", "ctbllib", "factint", "fga",
-              "irredsol", "laguna", "PackageManager", "polenta", "polycyclic",
+              "irredsol", "laguna", "polenta", "polycyclic",
               "resclasses", "sophus", "tomlib" ],
   values:= function() return RecNames( GAPInfo.PackagesInfo ); end,
   multi:= true,
-- 
2.51.0

